go.mod

```diff
@@ -74,7 +74,7 @@ require (
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	k8s.io/kube-openapi v0.0.0-20221012153701-172d655c2280 // indirect
-	k8s.io/kubernetes v1.25.4 // indirect
+	k8s.io/kubernetes v1.25.11 // indirect
```

Comment:
It would be good to start a Kubeturbo pod and do basic manual testing after these changes.

Reply:
Done. Description has been updated.
The pod was built, pushed to the private artifactory, deployed to ROSA and ran successfully.